sdk/monitor/azure-monitor-ingestion/src/test/java/com/azure/monitor/ingestion/LogData.java

```diff
@@ -3,21 +3,15 @@
 
 package com.azure.monitor.ingestion;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
-
 import java.time.OffsetDateTime;
 
 public class LogData {
 
-    @JsonProperty(value = "Time")
```

Comment:
This should be kept making sure interoperability with Jackson custom types is retained.